samples/bluetooth/ibeacon/src/main.c

```diff
@@ -59,7 +51,7 @@ static void bt_ready(int err)
 
 	/* Start advertising */
 	err = bt_le_adv_start(BT_LE_ADV_NCONN, ad, ARRAY_SIZE(ad),
-			      sd, ARRAY_SIZE(sd));
+			      NULL, NULL);
```

Comment:
Looks like CI is failing because of the above:
```
'bt_le_adv_start' makes integer from pointer without a cast [-Werror=int-conversion]
          NULL, NULL);
                ^~~~
In file included from /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/samples/bluetooth/ibeacon/src/main.c:12:
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/include/bluetooth/bluetooth.h:365:40: note: expected 'size_t' {aka 'unsigned int'} but argument is of type 'void *'
       const struct bt_data *sd, size_t sd_len);
                                 ~~~~~~~^~~~~~
```

Reply:
@jhedberg Sorry for the mistake, already fixed.